__tests__/components/CreateForm.test.jsx

```diff
@@ -0,0 +1,68 @@
+import '@testing-library/jest-dom'
+import { render, screen, fireEvent } from '@testing-library/react'
+import Page from '@/components/CreateForm'
+import { useSession } from 'next-auth/react';
+
+global.fetch = jest.fn();
+
+describe('Testing if you have access to creatForm component', () => {
+    const oneDay = 86400
+    const user = {
+        user: {
+            name: 'admin',
+            email: 'example@example.com',
+            image: 'example'
+        },
+        expires: new Date(Date.now() + oneDay).toISOString()
+    }
+    
+    const props = {
+        session: {
+            user: {
+                name: 'admin',
+                email: 'example@example.com',
+                image: 'example'
+            },
+        },
+        name: {
+            name: 'admin',
+            id: 'example'
+        }
+    }
+    it('Show form when logged in', () => {
+        useSession.mockReturnValue({
+            data: user,
+            status: 'authenticated'
+        })
+        render(<Page {...props}/>)
+        const title = screen.getByLabelText('Title')
+        const location = screen.getByLabelText('Location')
+        expect(title).toBeInTheDocument()
+        expect(location).toBeInTheDocument()
+    })
+
+    it('Do not show form when not logged in', () => {
+        const user = null
+        useSession.mockReturnValue({
+            data: user
+        })
+        render(<Page {...props}/>)
+        const noForm = screen.getByText('Please log in to create post')
+
+        expect(noForm).toBeInTheDocument()
+        expect(noForm).toHaveTextContent('Please log in to create post')
+    })
+
+    it('Check that it is submitted', async () => {
+        useSession.mockReturnValue({
+            data: user,
+            status: 'authenticated'
+        })
+        render(<Page {...props} />)
+        const button = screen.getByRole('button', {name: 'Submit'})
+        fireEvent.click(button)
+        const text = screen.getByText('Post created')
+        expect(text).toBeInTheDocument()
+        expect(text).toHaveTextContent('Post created')
```

Comment:
This doesn't seem needed given the check on line 64-65

Reply:
Doesn't look like you addressed this

Reply:
I have removed the `expect(text).toBeInTheDocument()` since it does similar checks in `const text = screen.getByText('Post created')`. I left `expect(text).toHaveTextContent('Post created')` in there since this check if the text content is in there